Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -350,6 +351,22 @@
             app:popUpTo="@id/mainFragment"
             app:popUpToInclusive="true" />
     </fragment>
+    <fragment
+        android:id="@+id/submissionDoneNoConsentFragment"
+        android:name="de.rki.coronawarnapp.ui.submission.fragment.SubmissionDoneNoConsentFragment"
+        android:label="SubmissionDoneNoConsentFragment">
+        <action
+            android:id="@+id/action_submissionDoneNoConsentFragment_to_mainFragment"
+            app:destination="@id/mainFragment"
+            app:popUpTo="@id/mainFragment"
+            app:popUpToInclusive="true" />
+
+        <action
+            android:id="@+id/action_submissionDoneNoConsentFragment_to_submissionSymptomIntroductionFragment"
+            app:destination="@id/submissionSymptomIntroductionFragment"
+            app:popUpTo="@id/submissionSymptomIntroductionFragment"
```

Comment:
Why do we need to pop the stack here?

Reply:
I believe once we navigate to Symptom introduction screen, we need not go back to the thank you screen. 
@harambasicluka  @d4rken please correct me if needed.

Reply:
Currently don't know.

Reply:
Your destination is submissionSymptomIntroductionFragment. That is not already on the stack at that moment, or is it? If you want to prevent from going back, than you need to pop the fragments that are on the stack, probable up to the home fragment.

Reply:
changes were made accordingly

Reply:
popUpTo and popUpToInclusive have no effect here. Please remove.